src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java

```diff
@@ -1232,7 +1232,7 @@ public RedisFuture<StreamScanCursor> hscan(KeyValueStreamingChannel<K, V> channe
 
     @Override
     public RedisFuture<StreamScanCursor> hscanNovalues(KeyStreamingChannel<K> channel, K key, ScanCursor scanCursor,
-                                                       ScanArgs scanArgs) {
+            ScanArgs scanArgs) {
```

Comment:
Is this according to the code style?

BTW I would like us to have some automatic code formatter applied so we do not have to manually check. Should we consider something like the [Maven formatter plugin](https://code.revelc.net/formatter-maven-plugin/)?

Reply:
using [formatting.xml](https://github.com/redis/lettuce/blob/main/formatting.xml) with vscode,  let me know if this works in different way with you setup.

Reply:
You're actually correct. This is the correct formatting.

Yet my question remains - I think we need to enforce this in the build steps, otherwise we need to make manual checks. I will put a task for myself.

Reply:
FWIW, there's a formatter XML and setting up your IDE to format on save/on commit is the best way to approach this. Installing any checkstyle rules imposes a heavy up-front work that doesn't catch everything and that creates a burden to contributors.

That being said, formatting issues should be ideally a symptom of an incorrect IDE setup. Any formatting caused by 3rd party PR's can be easily resolved during the merge when merging locally.